keanu-project/src/main/java/io/improbable/keanu/algorithms/variational/optimizer/gradient/GradientOptimizer.java

```diff
@@ -195,15 +160,10 @@ private double optimize(List<Vertex> outputVertices) {
 
         NonLinearConjugateGradientOptimizer optimizer;
 
-        try {
-            optimizer = new NonLinearConjugateGradientOptimizer(
-                updateFormula.apacheMapping,
-                new SimpleValueChecker(relativeThreshold, absoluteThreshold)
-            );
-        } catch (NotStrictlyPositiveException e) {
```

Comment:
This was put here to get more information on the Windows NPS issue we were seeing a few weeks ago. The issue wasn't actually with this piece of code but was the only place we could reproduce it. I've removed it because we don't need it and it might confuse someone in the future.